models/content_units_persons.go

```diff
@@ -22,7 +22,7 @@ import (
 // ContentUnitsPerson is an object representing the database table.
 type ContentUnitsPerson struct {
 	ContentUnitID int64 `boil:"content_unit_id" json:"content_unit_id" toml:"content_unit_id" yaml:"content_unit_id"`
-	PersonID      int64 `boil:"person_id" json:"person_id" toml:"person_id" yaml:"person_id"`
+	PersonID      int64 `boil:"person_gid" json:"person_id" toml:"person_id" yaml:"person_id"`
```

Comment:
I believe this is a typo since this is generated code and should never changed manually